tests/sentry/integrations/github/test_client.py

```diff
@@ -662,10 +659,8 @@ def test_fatal_integration(self, get_jwt):
         self.gh_client.integration = None
         with pytest.raises(Exception):
             self.gh_client.get_blame_for_file(self.repo, "foo.py", "main", 1)
-        buffer = IntegrationRequestBuffer(self.gh_client._get_redis_key())
-        assert buffer.is_integration_broken() is True
```

Comment:
I removed these because once an integration is detected to be broken, we clear the buffer meaning calls to this function will always return False.